.commitlintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["@commitlint/config-conventional"]
```

Comment:
do czego to sluzy?

Reply:
To jest taka paczka, która pozwala trzymać nazewnictwo commit-ów. 
Nie przepuści commit-a jeżeli nie będzie spełniało wymogów, np. przepuści feat: treść_commit-a, ale już nie przepuści coś: jakiś_tekst